app/test_main.py

```diff
@@ -1,3 +1,162 @@
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should return 0 for animal age less than 15",
+            ),
+            pytest.param(
+                -1,
+                -1,
+                [0, 0],
+                id="for negative values should return 0",
+            ),
+            pytest.param(
+                10 ** 1000,
+                10 ** 1000,
+                [
+                    2499999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999996,  # noqa: E501 Need loog number for check
+                    1999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999997  # noqa: E501 Need loog number for check
+                ],
+                id="Should work properly with large numbers",
+            )
+        ],
+    )
+    def test_get_human_age(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected: [int]
+    ) -> None:
+        assert (
+            get_human_age(cat_age, dog_age) == expected
+        )
```

Comment:
looks like it can be one lined
`assert get_human_age(cat_age, dog_age) == expected`